app/controllers/projects_controller.rb

```diff
@@ -155,7 +165,7 @@ def handle_unpublished_projects
       if current_user.logged_in?
         redirect_to chapter_project_path(current_project.chapter, current_project) and return
       else
-        render_404 and return
+        render_404 && return
```

Comment:
Just personal habit. It's what Rubocop and most linters will recommend as well, since it's easier to just use `&&` everywhere than remember the special cases where `and` isn't problematic. 😶